brunch-config.js

```diff
@@ -28,5 +34,14 @@ module.exports = {
         })
       ]
     }
+  },
+  sourceMaps: 'inline', // For karma-sourcemap-loader.
+  overrides: {
+    production: {
+      paths: {
+        // Exclude files in "test" from compilation.
+        watched: ['app', 'vendor']
```

Comment:
Yes, as the default set of directories in `paths.watched` is `['app', 'vendor', 'test']`, and the intent here was only to remove "test" from that set. Besides that, albeit unlikely, you may eventually wish to use a "vendor" directory for its intended purpose in the context of Brunch.